pkg/apb/deprovision.go

```diff
@@ -33,12 +35,17 @@ func Deprovision(instance *ServiceInstance, clusterConfig ClusterConfig, log *lo
 		"deprovision", clusterConfig, instance.Spec,
 		instance.Context, instance.Parameters, log,
 	)
-
```

Comment:
Eh, I probably absent-mindedly did this. Visually I think I like it because it associates the error handler with the line that triggered it.